### ISSUE TITLE
build error CS0246

### ISSUE THREAD

**Issue by 26PM** [keijiro/Lasp#26]
Excuse my ignorance if this is a simple problem to fix, I am a noob when it comes to scripting. When I try to build the project I get 24 errors with "error CS0246 regarding SoundIO. Example Packages\jp.keijiro.lasp\Runtime\Internal\InputDeviceHandle.cs(310,33): error CS0246: The type or namespace name 'SoundIO' could not be found (are you missing a using directive or an assembly reference?)

Is there a simple solution for this? Any help is appreciated. Thank you for making this awesome live sound project.

**Comment by keijiro** [comment]
How did you install the package? Could you re-check the "How to install" section in the README.md?

**Comment by 26PM** [comment]
YES! A fresh download, followed by modifying the scopedRegistries ( as specified in the How To Install directions), followed by opening the project as a new project fixed my problem. Thank you so much for the quick response. It is greatly appreciated. 

This may be helpful for other beginners. Originally I downloaded Lasp and ended up noticing while attempting to build that I needed to update Visual Studio. I updated Visual Studio but still had the Sound.IO errors. I modified the scopedRegistries  after the fact and this also did not fix the errors. I was able to build without errors after a fresh install as specified above.
